moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/tesseract/macros.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file macros.h
+ * @brief The tesseract_core package macros.
+ *
+ * @author Levi Armstrong
+ * @date April 15, 2018
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2013, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef TESSERACT_CORE_MACROS_H
+#define TESSERACT_CORE_MACROS_H
+#define TESSERACT_IGNORE_WARNINGS_PUSH                                                                                 \
+  _Pragma("GCC diagnostic push") _Pragma("GCC diagnostic ignored \"-Wall\"")                                           \
```

Comment:
If possible, just get of this file, it's doing some compiler magic with includes that should be handled in `CMakeLists.txt`/clang tidy configs.

Reply:
"get rid of", I assume

Reply:
Done